packages/keycodes/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@wordpress/keycodes",
 	"version": "1.0.1",
-	"description": "KeyCodes utilities for WordPress",
+	"description": "Keycodes utilities for WordPress",
```

Comment:
This one, I think, could be expanded upon further ("Keycodes utilities for WordPress; used to check for keyboard events across browsers/operating systems")